app/controllers/trn_submissions_controller.rb

```diff
@@ -7,13 +7,13 @@ def create
     RegisterForTrnJob.perform_later(trainee.id, current_user.dttp_id)
     RetrieveTrnJob.set(wait: RetrieveTrnJob::POLL_DELAY).perform_later(trainee.id)
 
-    redirect_to trn_submission_path(trainee_id: trainee.id)
+    redirect_to trn_submission_path(trainee_id: trainee.slug)
```

Comment:
Again, we can reduce this to just `trn_submission_path(trainee)`

Reply:
sorted in https://github.com/DFE-Digital/register-trainee-teachers/pull/405/commits/9e311434810f371cafa169c0e7559182064629a3